react-native-reanimated.d.ts

```diff
@@ -273,7 +273,16 @@ declare module 'react-native-reanimated' {
       getNode(): ReactNativeScrollView;
     }
     // eslint-disable-next-line @typescript-eslint/no-empty-interface
-    export interface ScrollView extends ReactNativeScrollView {}
+    export interface ScrollView extends ReactNativeScrollView {
+      currentProps: { horizontal: boolean };
+      _internalFiberInstanceHandleDEV: {
```

Comment:
Do we still need to modify the `ScrollView` type?

Reply:
It seems, that we don't need to do it anymore. Removed this part.